apps/web/src/pages/TalentNominations/NominationGroup/CareerExperience.tsx

```diff
@@ -188,7 +273,12 @@ const TalentNominationGroupCareerExperiencePage = () => {
   const { talentNominationGroupId } = useRequiredParams<RouteParams>(
     "talentNominationGroupId",
   );
-  const [{ data, fetching, error }] = useQuery({
+  const [{ data, fetching, error }] = useQuery<{
```

Comment:
Is the explicit type in the generic necessary here?

Reply:
otherwise I'm getting unsafe assignment type error in here  in the below line
```
  const nomineeId = data?.talentNominationGroup?.nominee?.id;
  const shareProfile = data?.talentNominationGroup?.consentToShareProfile;
```

Reply:
I see that, yeah.  If I restart eslint, though, it clears the error.  :shrug: